cedar-policy/Cargo.toml

```diff
@@ -14,6 +14,7 @@ repository = "https://github.com/cedar-policy/cedar"
 [dependencies]
 cedar-policy-core = { version = "=4.0.0", path = "../cedar-policy-core" }
 cedar-policy-validator = { version = "=4.0.0", path = "../cedar-policy-validator" }
+cedar-policy-formatter = { version = "=4.0.0", path = "../cedar-policy-formatter" }
```

Comment:
Did we have a reason to not take this dependency in the past?

Reply:
I don't believe so, @shaobo-he-aws do you remember any special reason?

Reply:
I think there are two reasons. 1. We only use the formatter in CLI so it's unnecessary there. 2. I was too lazy to add these APIs. So we're all good here.